src/components/icons/baseIcon/baseIcon.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC, MouseEventHandler, ReactElement, ReactNode } from 'react';
+import classNames from 'classnames/bind';
+import styles from './baseIcon.module.scss';
+
+const cx = classNames.bind(styles);
+
+export interface BaseIconProps {
+  children: ReactNode;
+  className?: string;
+  disabled?: boolean;
+  onClick?: MouseEventHandler<HTMLSpanElement>;
+}
+
+// DS link - https://www.figma.com/file/gjYQPbeyf4YsH3wZiVKoaj/%F0%9F%9B%A0-RP-DS-6?type=design&node-id=136-2&mode=design&t=Ri0jf5ugCvGn3z3b-0
```

Comment:
could you, please, advise if this comment is needed?

Reply:
I left links to a specific component in DS to speed up their search in Figma.
Later, I believe, these references will become part of the description of the storybook stories.